assets/src/containers/AssignmentPlanning.js

```diff
@@ -1,230 +0,0 @@
-import React, { useEffect, useState } from 'react'
-import { withStyles } from '@material-ui/core/styles'
-import Checkbox from '@material-ui/core/Checkbox'
-import FormControl from '@material-ui/core/FormControl'
-import Grid from '@material-ui/core/Grid'
-import MenuItem from '@material-ui/core/MenuItem'
-import Paper from '@material-ui/core/Paper'
-import Select from '@material-ui/core/Select'
-import Typography from '@material-ui/core/Typography'
-import { AssignmentPlanningTooltip } from '../components/Tooltip'
-import AlertBanner from '../components/AlertBanner'
-import ProgressBar from '../components/ProgressBar'
-import Spinner from '../components/Spinner'
-import TableAssignment from '../components/TableAssignment'
-import UserSettingSnackbar from '../components/UserSettingSnackbar'
-import ViewHeader from '../components/ViewHeader'
-import WarningBanner from '../components/WarningBanner'
-import useSetUserSetting from '../hooks/useSetUserSetting'
-import useUserSetting from '../hooks/useUserSetting'
-import { useAssignmentData } from '../service/api'
-import { getCurrentWeek } from '../util/data'
-import { isObjectEmpty } from '../util/object'
```

Comment:
It looks like these two utilities may not be used elsewhere and can be removed from those files.

Reply:
These are used in tests as of now. So I will leave it. `getCurrentWeek` is used with assignment V2.

Reply:
Hm, sorry, I must have missed those.